docs/navigation-structure.md

```diff
@@ -271,11 +271,14 @@ nav_external_links:
   - title: Just the Docs on GitHub
     url: https://github.com/just-the-docs/just-the-docs
     hide_icon: false # set to true to hide the external link icon - defaults to false
+    opens_in_new_tab: false # set to true to open this link in a new tab - defaults to false
 ```
 
 The external links are decorated by an icon, which distinguishes them from internal links.
 You can suppress the icon by setting `hide_icon: true`.
 
+You can set default global values for opening external links in new tabs by setting `nav_external_links_new_tab` to either `true` or `false`(default) in your site's `_config.yml` file. Setting the individual `opens_in_new_tab` on a link will override this value.
```

Comment:
What are your thoughts on this rewriting? Open to suggestions!

```suggestion
By default, external links are not opened in a new tab. However, this can be enabled by:

1. setting `opens_in_new_tab: true` in the link's configuration object
2. setting the configuration option `nav_external_links_new_tab: true` in `_config.yml`

When they conflict, `opens_in_new_tab` takes precedence.
```